### PR TITLE
Add some test utils

### PR DESCRIPTION
**Pull request by zguesmi**

### CODE REVIEW

File: src/main/java/com/iexec/common/utils/TestUtils.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import com.iexec.common.chain.ContributionAuthorization;
+import com.iexec.common.security.Signature;
+
+import org.web3j.utils.Numeric;
+
+public class TestUtils {
+
+    // pool credentials
+    public static final String POOL_ADDRESS = "0xc911f9345717ba7c8ec862ce002af3e058df84e4";
+    public static final String POOL_PRIVATE = "102522099393445372136873302035232363929034445624871422932329330474318520724961";
```
Comment (james-toussaint) [inline src/main/java/com/iexec/common/utils/TestUtils.java]: Can you put plain hex? (Much commonly used)

Comment (zguesmi) [reply src/main/java/com/iexec/common/utils/TestUtils.java]: Sure yes, thanks!

File: src/main/java/com/iexec/common/utils/TestUtils.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    // worker credentials
+    public static final String WORKER_ADDRESS = "0x87ae2b87b5db23830572988fb1f51242fbc471ce";
+    public static final String WORKER_PRIVATE = "0xd0db6df0ebcd1d41439d91d86c5fc5c1806ee9cd8e71e3d5544bb7294b435c26";
+    public static final String WORKER_PUBLIC = "0x76941b6e95be43ebfad0e81d7e2fae6268aa5f57e26cf3112adee8791d08775645"
+            + "a0c0879d7621ecc4f8c8b41b370ea0ffadd82693ffc429127fd6acd090f1ab";
+
+    public static final String TASK_ID = "0x1111111111111111111111111111111111111111111111111111111111111111";
+    public static final String DEAL_ID = "0x2222222222222222222222222222222222222222222222222222222222222222";
+    
+    public static final String NULL_ADDRESS         = "0x0000000000000000000000000000000000000000";
```
Comment (james-toussaint) [inline src/main/java/com/iexec/common/utils/TestUtils.java]: NULL_ADDRESS = BytesUtils.EMPTY_ADDRESS is you want :bulb:

File: src/main/java/com/iexec/common/utils/TestUtils.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    public static final String APP_ADDRESS          = "0xaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
+    public static final String REQUESTER_ADDRESS    = "0xbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb";
+    public static final String DATASET_ADDRESS      = "0xdddddddddddddddddddddddddddddddddddddddd";
+    public static final String ENCLAVE_ADDRESS      = "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee";
+
+    public static final String NON_TEE_TAG =    "0x0000000000000000000000000000000000000000000000000000000000000000";
+    public static final String TEE_TAG =        "0x0000000000000000000000000000000000000000000000000000000000000001";
```
Comment (james-toussaint) [inline src/main/java/com/iexec/common/utils/TestUtils.java]: TEE_TAG = TeeUtils.TEE_TAG (please put it public)

**Review by james-toussaint — APPROVED** [review]

File: src/main/java/com/iexec/common/utils/TestUtils.java (hunk 2 above)
Comment (zguesmi) [reply src/main/java/com/iexec/common/utils/TestUtils.java]: :+1:

**Pull request merged**